.jenkins

```diff
@@ -110,6 +110,7 @@ pipeline {
                                 -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
                                 -DKokkos_ENABLE_EXAMPLES=ON \
                                 -DKokkos_ENABLE_TESTS=ON \
+                                -DKokkos_ENABLE_BENCHMARKS=ON \
```

Comment:
Does it make sense to run it on our ci? as most of the machines do multiple things concurrently

Reply:
In an ideal world I would think they would not reveal anything the unit-tests do not already capture ... but given the world we live in

Reply:
This allows to keep the coverage at the same level, because current performance tests were enabled here (they are using common `Kokkos_ENABLE_TESTS` flag atm).

Reply:
I was just wondering if we want to stick with this behavior. If yes, it is fine by me